app/views/staff/level_b_activity_uploads/new.html.haml

```diff
@@ -0,0 +1,31 @@
+= content_for :page_title_prefix, t("page_title.activity.upload_level_b", organisation: @organisation.name)
```

Comment:
Very small preference for naming the string parameter passed to the locale `organisation_name` or `org_name`, but it's not a big deal.